src/ts/components/account/SelectNetwork.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
+import t from '../../services/i18n'
+import Progress from './Progress'
+import { withRouter, RouteComponentProps } from 'react-router'
+import {
+  Button as StyledButton,
+  ContentContainer,
+  TopSection
+} from '../basic-components'
+import { HOME_ROUTE } from '../../constants/routes'
+import { Divider, Form, Input } from 'semantic-ui-react'
+import styled from 'styled-components'
+import NetworkList from './NetworkList'
+
+interface ISelectNetworkProps extends RouteComponentProps {}
+
+interface ISelectNetworkState {
+  search: string
+}
+
+class SelectNetwork extends React.Component<ISelectNetworkProps, ISelectNetworkState> {
+
+  state = {
+    search: ''
+  }
+
+  gotoDashboard = () => {
+    this.props.history.push(HOME_ROUTE)
+  }
+
+  handleChange = (event) => {
+    this.setState({ search: event.target.value })
+  }
+
+  render () {
+
+    return (
+      <ContentContainer>
+        <TopSection>
+          <Progress step={3}/>
+        </TopSection>
+        <Form>
+          <ChainInput placeholder={'Chain Search'} onChange={this.handleChange}/>
+        </Form>
+        <NetworkSection>
+          <NetworkList search={this.state.search}/>
+        </NetworkSection>
+        <ChainQuote>Start by selecting <span>3 chains</span></ChainQuote>
+        <Divider />
+        <StyledButton type='button' onClick={this.gotoDashboard}>
```

Comment:
So the backend logic of saving to the store is not yet implemented, right?

Reply:
yes we need to discuss